addon/components/o-s-s/banner.stories.js

```diff
@@ -65,14 +66,25 @@ export default {
       }
     },
     disabled: {
-      description: 'Display the disabled version of the banner with background and text color updated',
+      description: 'Displays the disabled version of the banner with background and text color in grey',
       table: {
         type: { summary: 'boolean' },
         defaultValue: { summary: 'false' }
       },
       control: {
         type: 'boolean'
       }
+    },
+    size: {
+      description:
+        'Allows to adjust the size of the component. Currently available options are `sm` and `md`. Defaults to `md`.',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'false' }
+      },
+      control: {
+        type: 'string'
+      }
```

Comment:
```suggestion
    size: {
      description:
        'Allows to adjust the size of the component. Currently available options are `sm` and `md`. Defaults to `md`.',
      table: {
        type: SizeTypes.join('|'),
        defaultValue: { summary: 'md' }
      },
      options: SizeTypes,
      control: {
        type: 'select'
      }
```
Nit: You can also define `const SizeTypes = ['sm', 'md'];` and set a selection control type for the doc. Wdyt?

Reply:
for that little number of options you could also use a radio control type with`control: { type: 'radio' }`

Reply:
type radio does not change anything from the type: select on my side 🤷